owd.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new 'owd', OWD::VERSION do |s|
   s.extra_rdoc_files  = s.files.select { |p| p =~ /^README/ } << 'LICENSE'
   s.rdoc_options      = %w[--line-numbers --inline-source --title OWD --main README.rdoc]
 
-  s.add_dependency 'activesupport'
+  s.add_dependency 'activesupport', '~> 4.2.1'
```

Comment:
Yes, it was not locking the version and since 5.x is out, it was trying to install the latest version, which is not compatible with this gem. This locked it at 4.2.x.